.github/workflows/sbt-tasks.yml

```diff
@@ -0,0 +1,71 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: SBT Tasks
```

Comment:
I think "Tests" is fine. its nicely consistent with names of other actions.
Maybe we can call it "Unit tests" which is much meaningful

Reply:
its fine if it does additional non-unit-test type work. dont have to call it tasks.